src/main/java/com/google/idea/perf/Matching.kt

```diff
@@ -61,12 +83,38 @@ fun fuzzyMatch(source: String, pattern: String): MatchResult {
     val parentMatrix = Array(pattern.length + 1) { ByteArray(source.length + 1) }
 
     // Construct score and parent matrix
+    fun isDelimiter(c: Char) = c == '.' || c == '$' || c == '/'
+
+    fun getMatchScore(row: Int, column: Int): Int {
+        val char = source[column - 1]
+        val prevChar = source.getOrElse(column - 2) { ' ' }
+
+        return if (row == 1 && column == 1) {
```

Comment:
Optional: this likes a good use case for a 'when' expression rather than an 'if' expression:
```kotlin
return when {
    row == 1 && column == 1 -> FIRST_CHAR_SCORE
    prevChar.isLowerCase() && char.isUpperCase() -> CAMEL_CASE_SCORE
    isDelimiter(char) -> DELIMITER_SCORE
    isDelimiter(prevChar) -> POST_DELIMITER_SCORE
    else -> MATCH_SCORE
}
```

Reply:
Ooh nice, I will use this!